google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -316,6 +316,73 @@ def _create_load_subparser(parser):
   return load_parser
 
 
+def _create_pipeline_subparser(parser):
+  pipeline_parser = parser.subcommand('pipeline', 'Load from GCS, query, and extract data from GCS '
```

Comment:
I think the command description should be short single liner. Something like:

    Creates a pipeline to execute a SQL query to transform data using BigQuery.

Reply:
Fixed in https://github.com/googledatalab/pydatalab/pull/494